client/renderers/OpenGL/opengl.c

```diff
@@ -192,6 +197,9 @@ const char * opengl_get_name()
 static void opengl_setup()
 {
   option_register(opengl_options);
+  if (!getenv("WAYLAND_DISPLAY")) {
```

Comment:
I'm happy to move this logic into some sort of `is_wayland` helper, but am not sure where that should live. Perhaps `utils.c`?